docs/registry-temporary.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: docs
+title: "Using the temporary private registry"
+permalink: /docs/registry-temporary
+---
+
+# Working with MicroK8s' registry add-on
```

Comment:
The indentation here is bad here. In the demo page it is:
line 1:  Working with MicroK8s' registry add-
line 2: on

We could drop the "add-on" and say "Working with MicroK8s' registry".